app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,28 @@
+class OrdersController < ApplicationController
+
+  def new
+    @item_names = @cart.cart_items
+    @item_prices = @cart.cart_item_prices
+  end
+
+  def create
+    order = Order.new(order_date: Time.now)
```

Comment:
This was our finest moment last night! We can do some refactoring later but I think this setup for create is going to give us a lot of additional query capabilities for our admin dashboard.